### PR TITLE
Add custom ser/de examples

### PR DESCRIPTION
**Pull request by crajcan**

### CODE REVIEW

**Comment by sehz** [comment]
this should be merged into beta2 branch

**Comment by sehz** [comment]
LGTM

File: primitives/custom-serialization/struct/deserialize/README.MD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Map Operator Example
```
Comment (ajhunyady) [inline primitives/custom-serialization/struct/deserialize/README.MD:1]: Shouldn't the title say `Deserialization Example` ?

File: primitives/custom-serialization/struct/serialize/README.MD [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Map Operator Example
```
Comment (ajhunyady) [inline primitives/custom-serialization/struct/serialize/README.MD:1]: Title `Serialize Example` ?

File: primitives/custom-serialization/struct/serialize/dataflow.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      value:
+        type: citizen
+
+  citizens-public:
+    schema:
+      value:
+        type: citizen
+
+services:
+  map-service:
```
Comment (ajhunyady) [inline primitives/custom-serialization/struct/serialize/dataflow.yaml:33]: This is a `filter`, so it should be called `filter-service`.

File: primitives/custom-serialization/struct/deserialize/dataflow.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      value:
+        type: citizen
+
+  citizens-public:
+    schema:
+      value:
+        type: citizen
+
+services:
+  map-service:
```
Comment (ajhunyady) [inline primitives/custom-serialization/struct/deserialize/dataflow.yaml:33]: `filter-service` ?

**Review by ajhunyady — COMMENTED** [review]
@crajcan I know this was merged, but I found a few small items.

**Comment by sehz** [comment]
this was merged to beta2

**Comment by ajhunyady** [comment]
Sure, but there are a few issues with it.

**Comment by crajcan** [comment]
> Sure, but there are a few issues with it.
> 
> 

I'll fix this stuff in the other branch before it gets merged to main. Thanks for looking.

**Comment by ajhunyady** [comment]
awesome, thanks.

**Pull request merged**